File: Easy/Array/3289. The Two Sneaky Numbers of Digitville/Solution.java
class Solution {
    public int[] getSneakyNumbers(int[] nums) {

        int[] res = new int[2];

        Set<Integer> set = new HashSet<Integer>();

        int i = 0;

        for(int n: nums){

            if(set.contains(n)){
                res[i++] = n;
            }
            
            set.add(n);
        }

        return res;
    }
}

File: Easy/Array/3289. The Two Sneaky Numbers of Digitville/Solution.kt
class Solution {
    fun getSneakyNumbers(nums: IntArray): IntArray {
        
        val set = mutableSetOf<Int>()
        val result = IntArray(2)

        var i = 0

        for(num in nums) {
            if(!set.add(num)){
                result[i++] = num
            }
        }

        return result
    }
}
